.github/workflows/stale.yml

```
@@ -12,10 +12,14 @@ jobs:
     steps:
       - uses: actions/stale@v9
         with:
+          stale-issue-message: 'This issue was marked stale due to lack of activity and will be closed in 7 days. Commenting will instruct the bot to automatically remove the label. This bot runs once per day.'
+          close-issue-message: 'Closed as inactive. Feel free to reopen if this issue is still a concern.'
           stale-pr-message: 'This PR was marked stale due to lack of activity and will be closed in 7 days. Commenting or Pushing will instruct the bot to automatically remove the label. This bot runs once per day.'
           close-pr-message: 'Closed as inactive. Feel free to reopen if this PR is still being worked on.'
           operations-per-run: 400
           days-before-pr-stale: 7
-          days-before-issue-stale: -1
+          days-before-issue-stale: 1200
           days-before-pr-close: 7
-          days-before-issue-close: -1
+          days-before-issue-close: 7
+          exempt-all-issue-milestones: true
+          exempt-issue-labels: need-runtime-change,await-spec,help wanted
```

```suggestion
          exempt-issue-labels: need-runtime-change,await-spec,help wanted,needs-triage 
```

PS: I just created `needs-triage` label: https://github.com/open-telemetry/opentelemetry-dotnet/labels?q=needs-triage